lessonbuilder/api/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDao.java

```diff
@@ -196,6 +196,8 @@
 
 	public SimplePage getPage(long pageId);
 
+	public String getPageUrl(long pageId);
```

Comment:
I'm not sure that `SimplePageToolDao` is a good place to put this as I would expect that there would be a service to expose this sort of functionality and the DAO is hidden behind the service.